dask/dataframe/tests/test_dataframe.py

```diff
@@ -2355,7 +2355,7 @@ def test_delayed_roundtrip(optimize: bool):
     for x in delayed2:
         x.dask.validate()
 
-    df3 = dd.from_delayed(delayed2, meta=df1, divisions=df1.divisions)
+    df3 = dd.from_delayed(delayed2, meta=df1, divisions=df1.divisions)  # type: ignore
```

Comment:
Can you add a TODO` to type `from_delayed`? This papercut is annoying enough that I'd like to try to fix it at some point